certified-connectors/Mural/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1074 @@
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [],
+  "paths": {
+    "/api/public/v1/murals": {
+      "post": {
+        "responses": {
+          "201": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "value": {
+                  "type": "object",
+                  "properties": {
+                    "backgroundColor": {
+                      "type": "string",
+                      "description": "Background Color"
+                    },
+                    "createdBy": {
+                      "type": "object",
+                      "properties": {
+                        "email": {
+                          "type": "string",
+                          "description": "email"
+                        },
+                        "firstName": {
+                          "type": "string",
+                          "description": "firstName"
+                        },
+                        "id": {
+                          "type": "string",
+                          "description": "id"
+                        },
+                        "lastName": {
+                          "type": "string",
+                          "description": "lastName"
+                        },
+                        "type": {
+                          "type": "string",
+                          "description": "type"
+                        }
+                      },
+                      "description": "createdBy"
+                    },
+                    "createdOn": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "createdOn"
+                    },
+                    "embedLink": {
+                      "type": "string",
+                      "description": "embedLink"
+                    },
+                    "favorite": {
+                      "type": "boolean",
+                      "description": "favorite"
+                    },
+                    "height": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "height"
+                    },
+                    "id": {
+                      "type": "string",
+                      "description": "id"
+                    },
+                    "roomId": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "roomId"
+                    },
+                    "sharingSettings": {
+                      "type": "object",
+                      "properties": {
+                        "link": {
+                          "type": "string",
+                          "description": "link"
+                        }
+                      },
+                      "description": "sharingSettings"
```

Comment:
@dmytroIvanovDev **All the description values of properties in the swagger** are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
@dmytroIvanovDev This needs to be addressed and resolved.